transform/snowflake-dbt/models/blp/license_server_fact.sql

```diff
@@ -12,109 +12,14 @@
 
 {% endif %}
 
-with account_mapping as (
-  SELECT 
-      elm.account_sfid
-    , a.name as account_name
-    , elm.licenseid as license_id
-    , elm.opportunity_sfid
-    , elm.company
-    , elm.contact_sfid
-    , elm.edition
-  FROM (
-        SELECT
-            COALESCE(elm.account_sfid, lo.account_sfid)         AS account_sfid
-          , COALESCE(elm.opportunity_sfid, lo.opportunity_sfid) AS opportunity_sfid
-          , COALESCE(trim(elm.licenseid), trim(lo.licenseid))   AS licenseid
-          , COALESCE(trim(elm.company), trim(lo.company))       AS company
-          , COALESCE(trim(lo.contact_sfid), NULL)       AS contact_sfid
-          , lo.edition AS edition
-        FROM {{ ref('enterprise_license_mapping') }} elm
-        FULL OUTER JOIN {{ ref('license_overview') }} lo
-          ON trim(elm.licenseid) = trim(lo.licenseid)
-        GROUP BY 1, 2, 3, 4, 5, 6
-      ) elm
-  LEFT JOIN {{ ref( 'account') }} a
-      ON elm.account_sfid = a.sfid
-  GROUP BY 1, 2, 3, 4, 5, 6, 7
-),
-
-licensed_servers as (
-SELECT
-    {{ dbt_utils.surrogate_key(['l.license_id', 'l.server_id']) }} as id
```

Comment:
We don't need to add account_mapping since the unique ID is being generated in licensed_servers and nonactivated_licenses. @jasonblais